TerminalDocs/customize-settings/profile-appearance.md

```diff
@@ -91,9 +91,9 @@ This sets the weight (lightness or heaviness of the strokes) for the profile's f
 }
 ```
 
-### Intense Text Formatting
+### Intense text formatting ([Preview](https://aka.ms/terminal-preview))
```

Comment:
I actually backported this one to stable, sorry. It was either this or to leave bold=bold on by default which people would have hated b/c of the rendering bugs

Reply:
No worries, I'll fix it now :)